src/conn.rs

```diff
@@ -17,13 +17,17 @@ pub fn handle(mut stream: TcpStream) {
         Ok((user, pw)) => auth::find_user(&user, &pw),
         _ => Err(auth::AuthError::UserNotFound)
     };
-        
-    // TODO: Read commands from the client (with help of `net`)
 
+    // TODO: Read commands from the client (with help of `net`)
+    loop {
+        //get the command from the stream
+        let command_res = net::read_commands(&mut stream);
+        
```

Comment:
What about `println!("{:?}", command_res);`? It would eliminate the "unused variable" warning and we would see the debug output of the command.